.gitignore

```diff
@@ -6,3 +6,5 @@ __pycache__
 .vscode/
 # Ignore Mac system files
 .DS_Store
+# Ignore devcontainer for now (should this go in source control?)
+.devcontainer
```

Comment:
Not sure what this is—so I'm not sure if it belongs in source control. 😇

Reply:
I see. .devcontainer is a vscode feature which defines a container you can do development in. Some projects (including home assistant) include this in source control so that developers have a consistent development environment. I develop in a devcontainer, so I have it in the gitignore so it doesn't get added when I do a `git add .`. 

Is this something you'd be interested in? Otherwise I'll just figure out how to get git to skip it without polluting the repositories' gitignore.